OloEditor/src/OloEditorApp.cpp

```
@@ -8,24 +8,24 @@
 
-		~OloEngineEditor() final = default;
-	};
+        ~OloEngineEditor() final = default;
+    };
 
-	Application* CreateApplication(ApplicationCommandLineArgs const args)
-	{
-		ApplicationSpecification spec;
-		spec.Name = "OloEditor";
-		spec.CommandLineArgs = args;
+    Application* CreateApplication(ApplicationCommandLineArgs const args)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)